File: Kotlin/ROUND_2_B/src/main/kotlin/Game.kt
class Game(val homeTeam: String, val homeTeamScore: Int, val awayTeam: String, val awayTeamScore: Int) {

    fun teamScore(teamName: String, total: Int): Int {
        var total1 = total
        if (this.homeTeam.equals(teamName)) {
            total1 += this.homeTeamScore
        }
        if (this.awayTeam.equals(teamName)) {
            total1 += this.awayTeamScore
        }
        return total1
    }
}

File: csharp/ROUND_2/ROUND_2_A/src/Game.cs
namespace ROUND_2_A;

public class Game
{
    public string HomeTeam { get; }
    public int HomeTeamScore { get; }
    public string AwayTeam { get; }
    public int AwayTeamScore { get; }

    public Game(string homeTeam, int homeTeamScore, string awayTeam, int awayTeamScore)
    {
        HomeTeam = homeTeam;
        HomeTeamScore = homeTeamScore;
        AwayTeam = awayTeam;
        AwayTeamScore = awayTeamScore;
    }
}